src/app/modules/submission/encrypt-submission/encrypt-submission.controller.ts

```diff
@@ -367,9 +369,23 @@ const submitEncryptModeForm: ControllerHandler<
       })
     }
 
+    const paymentReceiptEmail = req.body.paymentReceiptEmail
+    if (!paymentReceiptEmail) {
+      logger.error({
+        message:
+          'Error when creating payment: payment receipt email not provided.',
+        meta: logMeta,
+      })
+      return res.status(StatusCodes.BAD_REQUEST).json({
+        message:
+          "The form's payment settings are invalid. Please contact the admin of the form to rectify the issue.",
```

Comment:
Should this error message be more specific to say the the payment receipt email cannot be found? Because form admins will still not know what is wrong given this error message by respondents.
Maybe this needs some design input?

Reply:
haha i just reused the same error message from above, we can change if necessary later on!

Reply:
aights! should be okay for now since we have different error codes 👍